### ISSUE TITLE
Polyglot archetype - Cannot package nodejs apps with runtime greater than version 14 of nodejs

### ISSUE THREAD

**Issue by IoannisF-NRB** [vmware/build-tools-for-vmware-aria#388]
### Description
We're using Aria Orchestrator version 8.18 and try to implement a nodejs action through polyglot with the build tools version 2.41.1 to be able to have a small package with some specific nodejs dependencies in it (such as webdav). 

### Steps to Reproduce

1. I've first tried to package it letting all polyglot.json config by default but when I push the package into Aria Orchestrator, it doesn't work as it has a node:12 runtime (And this is deprecated. See [the documentation page] (https://docs.vmware.com/en/VMware-Aria-Automation/8.18/Using-Automation-Orchestrator/GUID-13E3704E-A325-4CB1-AF57-45FEBDBBD159.html) )
2. I've tries to force the runtime version by adding a custom environment and specifying the version of node (node:20) but it failed to package the solution because the action runtime specified is not yest supported 
![image](https://github.com/user-attachments/assets/b74a9559-f7a5-45dd-8e6e-256835bdefd9)

**Preconditions:**
Create a polyglot project with maven using the polyglot-archetype for vro (See [doc](https://github.com/vmware/build-tools-for-vmware-aria/blob/main/docs/versions/latest/Components/Archetypes/Polyglot/General/New%20Project.md))

**Expected behavior:**
I should be able to package the solution with the new versions of nodejs's Aria Orchestrator runtime

**Actual behavior:**
Cannot package the solution for newer version of Aria Orcherstrator

**Reproduces how often:** [What percentage of the time does it reproduce]
100% of the time

**Component/s:**
typescript/polyglotpkg

**Affects Build/s:** [Which are the Build Tools for VMware Aria releases / builds affected by the issue]

#### Client
-   Build Tools for VMware Aria Version: 2.41.1
-   Visual Studio Code Version: 1.92.2
-   OS Version: /

#### Server
-   vRealize Automation Version: /
-   vRealize Orchestrator Version:  8.18.0.24015865
-   vRealize Operations Version: /
-   vRealize Log Insight Version: /

### Failure Logs
See the picture above in the step to reproduce

### Related issues and PRs
/

### Additional Context
/

**Comment by Michaelpalacce** [comment]
Would be nice if we provided somewhere the documentation of what is supported....

```js
    VRO_NODEJS_12 = 'node:12',
    VRO_NODEJS_14 = 'node:14',
    VRO_POWERCLI_11_PS_62 = 'powercli:11-powershell-6.2',
    VRO_POWERCLI_12_PS_71 = 'powercli:12-powershell-7.1',
    VRO_PYTHON_37 = 'python:3.7',
    VRO_PYTHON_310 = 'python:3.10',
    ABX_NODEJS = 'nodejs',
    ABX_POWERSHELL = 'powershell',
    ABX_PYTHON = 'python',
```

that being said these are the options for the runtime

it's part of the `platform` section 

```json
{
...
"platform": {
 "runtime" : "node:14"
}

}
```

**Comment by IoannisF-NRB** [comment]
NodeJS version 12 and 14 are not available anymore for Aria Orchestration 8.18. The only supported versions are Node 18 and 20.. Is it possible to fix the PolyglotPackage so it can handle those 2 environments to be compliant with version 8.18 of Aria Orchestrator ? 
https://docs.vmware.com/en/VMware-Aria-Automation/8.18/Using-Automation-Orchestrator/GUID-13E3704E-A325-4CB1-AF57-45FEBDBBD159.html

![image](https://github.com/user-attachments/assets/6c9e3c42-3790-4b65-80f1-a5589eb4afbb)

**Comment by Michaelpalacce** [comment]
Should be pretty simple. Thanks for pointing that out
